content/post/manage-mongodb-in-aks/index.md

```diff
@@ -0,0 +1,581 @@
+	"user" : "root",
+	"db" : "admin",
+	"roles" : [
+		{
+			"role" : "root",
+			"db" : "admin"
+		}
+	],
+	"mechanisms" : [
+		"SCRAM-SHA-1",
+		"SCRAM-SHA-256"
+	]
+}
+
+rs:PRIMARY> use musicdb
+switched to db musicdb
+
+rs:PRIMARY> db.bands.insert({"name":"Backstreet Boys"});
+WriteResult({ "nInserted" : 1 })
+
+rs:PRIMARY> db.bands.find().pretty()
+{ "_id" : ObjectId("623aa7550d970b42f91471a3"), "name" : "Backstreet Boys" }
+
+rs:PRIMARY> exit
+bye
+
+```
+
+> This was just an example of our MongoDB ReplicaSet database deployment. The other databases that KubeDB supports are Elasticsearch, Redis, MySQL, PostgreSQL, MariaDB, Percona XtraDB, ProxySQL, Memcached and PgBouncer. More information about Run & Manage MongoDB on Kubernetes can be found [HERE](https://kubedb.com/kubernetes/databases/run-and-manage-mongodb-on-kubernetes/)
+
+## Backup MongoDB Database Using Stash
+
+Here, we are going to use Stash to backup the MongoDB database we deployed before.
+
+### Step 1: Install Stash
+
+Go to [Appscode License Server](https://license-issuer.appscode.com/) again to get the Stash Enterprise license.
+Here, we will use the Stash Enterprise license that we obtained.
+
+```bash
+$ helm install stash appscode/stash             \
+  --version v2022.02.22                         \
+  --namespace kube-system                       \
+  --set features.enterprise=true                \
+  --set-file global.license=/path/to/the/license.txt
+```
+
+Let's verify the installation:
+
+```bash
+$ watch kubectl get pods --all-namespaces -l app.kubernetes.io/name=stash-enterprise
+NAMESPACE     NAME                                      READY   STATUS    RESTARTS   AGE
+kube-system   stash-stash-enterprise-786687ff94-j5gnb   2/2     Running   0          10m
+```
+
+Now, to confirm CRD groups have been registered by the operator, run the following command:
+
+```bash
+$ kubectl get crd -l app.kubernetes.io/name=stash
+NAME                                      CREATED AT
+backupbatches.stash.appscode.com          2022-03-23T05:31:28Z
+backupblueprints.stash.appscode.com       2022-03-23T05:31:29Z
+backupconfigurations.stash.appscode.com   2022-03-23T05:31:27Z
+backupsessions.stash.appscode.com         2022-03-23T05:31:27Z
+functions.stash.appscode.com              2022-03-23T05:29:56Z
+repositories.stash.appscode.com           2022-03-22T06:49:21Z
+restorebatches.stash.appscode.com         2022-03-23T05:31:29Z
+restoresessions.stash.appscode.com        2022-03-22T06:49:21Z
+tasks.stash.appscode.com                  2022-03-23T05:29:56Z
+
+```
+
+
+### Step 2: Prepare Backend
+
+Stash supports various backends for storing data snapshots. It can be a cloud storage like GCS bucket, AWS S3, Azure Blob Storage etc. or a Kubernetes persistent volume like HostPath, PersistentVolumeClaim, NFS etc.
+
+For this tutorial we are going to use gcs-bucket. You can find other setups [here](https://stash.run/docs/v2022.02.22/guides/backends/overview/).
```

Comment:
As we are using Azure, we should've done it using Azure instead of Google cloud.

Reply:
fixed